src/Helpers/babylon.videoDome.ts

```diff
@@ -0,0 +1,58 @@
+module BABYLON {
+    /**
+     * Display a 360 degree video on an approximately spherical surface, useful for VR applications or skyboxes, video attempts to autoplay.
+     * Subclass of Node so there can be multiple locational videos or instead have it parented to the camera.
+     * This effect can be achieved by using a VideoTexture and correctly configured material on an inverted sphere.
+     * Potential additions to this helper include zoom and and non-infinite distance rendering effects.
+     */
+    export class VideoDome extends Node {
+
+        /**
+         * The video texture being displayed on the sphere
+         */
+        private _videoTexture: VideoTexture;
+
+        /**
+         * The skybox material
+         */
+        private _material: StandardMaterial;
+
+        /**
+         * The surface used for the skybox
+         */
+        private _mesh: Mesh;
+
+        constructor(name: string, urlsOrVideo: string[] | HTMLVideoElement, options: {
+            clickToPlay?: boolean,
+            size?: number
+        }, scene: Scene) {
+            super(name, scene);
+
+            // set defaults and manage values
+            name = name || "videoDome";
+            options.clickToPlay = Boolean(options.clickToPlay);
+            options.size = Math.abs(options.size as any) || 1000;

```

Comment:
1000 should be deduced from camera clipbplanes

Reply:
How would I look up the current camera to make this determination? The scene does not seem to have an active camera property, the other option would be hard coupling the helper to a specific camera but I'd like to avoid that.

Reply:
It's there in the live docs, but not the local version along with many other things. Hrm, I might have more issues with my local documentation compile than I thought. Will investigate.

Reply:
What do you mean by local documentation compile? The Babylon.d.ts file?

Reply:
The compile of the documentation I've done locally on my system in order to test the documentation I'm writing in the class.

Reply:
Ok. we've just pushed a new gulp task for that: gulp typedoc-check